styles/globals.css

```diff
@@ -7,10 +7,18 @@
 	padding: 0;
 	box-sizing: border-box;
 	scroll-behavior: smooth;
+	
 }
 
 body {
 	@apply text-lg leading-8 text-primary-black;
+	margin: 5%;
```

Comment:
Was the project working fine with others code when added these changes?

Reply:
Yes , and I never touch global.css